codelists/financingArrangementType.csv

```diff
@@ -0,0 +1,8 @@
+Code,Title,Description
+bond,Bond,"A debt-based financial instrument issued by a national government, subnational government, project company or corporate entity. A bond typically obliges the issuer to make interest payments (coupons) over a specified period and to repay the principle (amount borrowed) at the maturity date."
```

Comment:
We use "equity instrument". Can we use "debt instrument" instead of "debt-based financial instrument" for bond and loan?

Reply:
Yep, that makes sense. Done in https://github.com/open-contracting-extensions/ocds_finance_extension/pull/26/commits/8d233279f907ff4b0ae4182ce35587e5f6219e4f